### PR TITLE
added more utility functions to Random

### PR DESCRIPTION
**Pull request by minhnhdo**

### CODE REVIEW

**Comment by magnus-madsen** [comment]
@minhnhdo Thanks for your contribution and welcome to little community!

Could I ask you to add your name to AUTHORS.md to make clear this is open source? Thanks!

File: main/src/library/Random.flix [hunk 1]
```diff
@@ -58,6 +58,26 @@ mod Random {
...
+
+    ///
+    /// Returns the next pseudorandom 16-bit integer value from the given random number generator `r`.
+    ///
+    pub def nextInt16(r: Random): Int16 \ IO =
+        import static java.lang.Integer.valueOf(Int32): ##java.lang.Integer \ {};
+        import java.lang.Integer.shortValue(): Int16 \ {};
+        nextInRange(r, minInclusive = Int16.toInt32(Int16.minValue()), maxExclusive = Int16.toInt32(Int16.maxValue()) + 1)
+        |> valueOf
+        |> shortValue
```
Comment (magnus-madsen) [inline main/src/library/Random.flix:79]: Any reason not to just create a random Int32 and then immediately convert it to an Int16? 

I think that would be simpler.

Comment (minhnhdo) [reply main/src/library/Random.flix:79]: Let me preface that I'm a random number generation noob.

> Any reason not to just create a random Int32 and then immediately convert it to an Int16?

I don't know if doing that would make the generated pattern no longer uniform. Do you know?

**Comment by magnus-madsen** [comment]
I am closing this for now. Let me know if you want to continue it in the future.

**Pull request closed without merging**